io/plugins/eu.esdihumboldt.hale.io.jdbc.test/src/eu/esdihumboldt/hale/io/jdbc/test/AbstractDBTest.java

```diff
@@ -94,15 +113,20 @@ public void setupDB() throws DockerException, InterruptedException {
 			Thread.currentThread().setContextClassLoader(cl);
 		}
 
-		jdbcUri = URI.create(dbi.getJDBCURL(client.getHostPort(dbi.getDBPort()),
-				client.getHostName()));
-
+		jdbcUri = URI.create(dbi.getJDBCURL(
+				client.getHostPort(dbi.getDBPort()), client.getHostName()));
+		jdbcUri = URI.create("jdbc:oracle:thin:@//localhost:1521/demoDB");
+		TestUtil.startConversionService();
+		TestUtil.startService(
+				Arrays.asList("eu.esdihumboldt.hale.io.jdbc.oracle"),
```

Comment:
This would have to be done in an Oracle specific test class.
Also, it fails if the bundle is not available (which is the case during tests on the build server).